srcpkgs/chirp/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'chirp'
+pkgname=chirp
+version=20190524
+revision=1
+build_style=python2-module
```

Comment:
We don't allow ne Python 2 only packages, if chirp works with py3 than update this PR

Reply:
setup.py wouldn't even run with py3. Figured it was probably a safe assumption the rest would be a similar story. Trying run run the included `chirpw` script throws a SyntaxError.

```
=> chirp-20190524_1: running do_build ...
make: Entering directory '/builddir/chirp-daily-20190524/locale'
make: *** No rule to make target 'clean'.  Stop.
make: Leaving directory '/builddir/chirp-daily-20190524/locale'
LOC: [('share/chirp/locale/pt_BR/LC_MESSAGES', ['locale/pt_BR/LC_MESSAGES/CHIRP.mo']), ('share/chirp/locale/en_US/LC_MESSAGES', ['locale/en_US/LC_MESSAGES/CHIRP.mo']), ('share/chirp/locale/es_ES/LC_MESSAGES', ['locale/es_ES/LC_MESSAGES/CHIRP.mo']), ('share/chirp/locale/it/LC_MESSAGES', ['locale/it/LC_MESSAGES/CHIRP.mo']), ('share/chirp/locale/fr/LC_MESSAGES', ['locale/fr/LC_MESSAGES/CHIRP.mo']), ('share/chirp/locale/uk_UA/LC_MESSAGES', ['locale/uk_UA/LC_MESSAGES/CHIRP.mo']), ('share/chirp/locale/ru/LC_MESSAGES', ['locale/ru/LC_MESSAGES/CHIRP.mo']), ('share/chirp/locale/nl/LC_MESSAGES', ['locale/nl/LC_MESSAGES/CHIRP.mo']), ('share/chirp/locale/de/LC_MESSAGES', ['locale/de/LC_MESSAGES/CHIRP.mo']), ('share/chirp/locale/hu/LC_MESSAGES', ['locale/hu/LC_MESSAGES/CHIRP.mo']), ('share/chirp/locale/pl/LC_MESSAGES', ['locale/pl/LC_MESSAGES/CHIRP.mo'])]
running build
running build_py
running build_scripts
  File "setup.py", line 17
    print "Set chirp/__init__.py::__all__ = %s" % str(chirp.__all__)
                                              ^
SyntaxError: invalid syntax
```

Reply:
The this will have to wait until upstream releases a version that works with python 3.